python/tvm/relay/op/strategy/cuda.py

```diff
@@ -222,12 +218,11 @@ def conv2d_strategy_cuda(attrs, inputs, out_type, target):
 
             # register auto-scheduler implementations
             if judge_winograd_auto_scheduler:
-                strategy.add_auto_scheduler(
-                    wrap_compute_conv2d(topi.nn.conv2d_winograd_nhwc), name="conv2d_nhwc.winograd"
-                )
-            else:
-                strategy.add_auto_scheduler(
-                    wrap_compute_conv2d(topi.nn.conv2d_nhwc), name="conv2d_nhwc"
+                strategy.add_implementation(
```

Comment:
This plevel is the same as `conv2d_nhwc.cuda`, which one will auto-scheduler pick?
which one will autotvm pick?